TombEngine/Specific/level.h

```diff
@@ -150,7 +157,7 @@ struct LEVEL
 	std::vector<ANIMATED_TEXTURES_SEQUENCE> AnimatedTexturesSequences = {};
 };
 
-extern const std::vector<GAME_OBJECT_ID> BRIDGE_OBJECT_IDS;
```

Comment:
Object ID is a well-known concept in TR source code. For example, we have `GAME_OBJECT_ID` enum. No need to rename it.

Reply:
I've named it back for now, though I have an idea for how to consistently enshrine various terminology of object concepts in the future.

Reply:
I am not sure there is need in changing existing terminology, if it is well-known and it has a clear meaning.

Reply:
Item/moveable?
Static/mesh?
Object/slot?

These are not clear-meaning. It's not changing terminology, but rather enforcing what is already standard outside the C++ implementations.

Reply:
These have clear meaning and can be interchanged, it's called synonyms. There is nothing wrong with it.

Reply:
Vastly different synonyms lead to error and are not good. There is everything wrong with it.

Reply:
There was no error in TEN source code originating from current terminology. Therefore, there is nothing wrong with it.

Reply:
I don't care anymore.